src/main/java/com/marvinformatics/hibernate/json/JsonUserType.java

```diff
@@ -79,7 +78,14 @@ public void nullSafeSet(PreparedStatement st, Object value, int index, SessionIm
         st.setObject(index, dataObject);
     }
 
-    @Override
+    public Object nullSafeGet(ResultSet rs, String[] names, SharedSessionContractImplementor session, Object owner) throws HibernateException, SQLException {
+        return nullSafeGet(rs, names, (SessionImplementor) session, owner);
```

Comment:
I don't think this cast is either safe nor necessary.

Reply:
right. I blindly copied the fix from my other project, which has jsonb working with hibernate using sprint boot 2.0.1. ill update this.